recipe/build-r-base.sh

```diff
@@ -356,6 +357,8 @@ Mingw_w64_makefiles() {
         sed -i 's|R_INSTALLER_BUILD = yes|R_INSTALLER_BUILD = no|g' ${_makeconf}
     done
 
+    rm -rf ${PREFIX}/lib/R/share/zoneinfo
```

Comment:
Would it be fine to use the mingw-provided `time.h`?
Could there be assumptions for R on Windows that expect the code to always be there?
If not, would it make sense to patch out building it?
(`tzone` in `EXTRALIBS`/`rlibs`/`EXTRA_DIRS` `in src/gnuwin32/Makefile` and `USE_INTERNAL_MKTIME` in `src/gnuwin32/fixed/h/config.h`)

Reply:
>  Could there be assumptions for R on Windows that expect the code to always be there?

That's okay I think. We can revert this PR in the unlikely case that there are issues.

Reply:
Wait, you were talking about `time.h`. Not sure

Reply:
Yep, I'm essentially asking if it would be okay to do a `--without-internal-tzcode` equivalent and unset `USE_INTERNAL_MKTIME` so that it'd use mingw's `time.h` instead of internal code.

Reply:
I don't know if it would be okay or not.

Reply:
Okay, then let's leave this as is for now.